prog/postgres/postgres_server_nexus.rb

```diff
@@ -156,6 +198,30 @@ def before_run
     hop_wait
   end
 
+  label def wait_recovery_completion
+    is_in_recovery = vm.sshable.cmd("sudo -u postgres psql -At -c 'SELECT pg_is_in_recovery()'").chomp == "t"
+
+    if is_in_recovery
+      is_wal_replay_paused = vm.sshable.cmd("sudo -u postgres psql -At -c 'SELECT pg_get_wal_replay_pause_state()'").chomp == "paused"
+      if is_wal_replay_paused
+        vm.sshable.cmd("sudo -u postgres psql -c 'SELECT pg_wal_replay_resume()'")
```

Comment:
could do this, could (follow-up software engineering idea) add `pg` to Rhizome and do Real Queries via febe protocol and unix socket.

In pure play Postgres services I would also have suggested using network tunnels, but for our more diverse set of services and your handy fragmented rhizome refactor, it seems like including relevant drivers in each service in question is more sensible, rather than forcing the control plane to be a union (as would be the case in tunnels).

This could be a progressive thing, since it may be easier to mess around with stuff if you don't need an `install_rhizome` step. So, there's another thought for engineering fluency.